bigchaindb/backend/connection.py

```diff
@@ -40,6 +40,12 @@ def connect(backend=None, host=None, port=None, name=None, replicaset=None):
     host = host or bigchaindb.config['database']['host']
     port = port or bigchaindb.config['database']['port']
     dbname = name or bigchaindb.config['database']['name']
+    # Not sure how to handle this here. This setting is only relevant for
+    # mongodb.
+    # I added **kwargs for both RethinkDBConnection and MongoDBConnection
+    # to handle these these additional args. In case of RethinkDBConnection
+    # it just does not do anything with it.
+    replicaset = replicaset or bigchaindb.config['database'].get('replicaset')
```

Comment:
> I added **kwargs

👍 , I think this is what we should be doing going forward.

Since we're only planning to add `replicaset` at this point, I passing this value into every `Connection` is fine for now.

The changes I made in https://github.com/bigchaindb/bigchaindb/pull/986 moved towards allowing arbitrary configuration properties (like `replicaset`), but it's unlikely we'll be able to have a nice way to do this until we rework the configuration at some point.